e2e/cli-e2e/tests/__snapshots__/compare.report-diff.md

```diff
@@ -4,37 +4,35 @@
 
-|🔌 Plugin|🗃️ Group|⭐ Previous score|⭐ Current score|🔄 Score change|
-|:--|:--|:--:|:--:|:--:|
-|[ESLint](https://www.npmjs.com/package/@code-pushup/eslint-plugin)|Suggestions|🟡 50|🟡 **71**|![↑ +21.4](https://img.shields.io/badge/%E2%86%91%20%2B21.4-green)|
+| 🔌 Plugin                                                          | 🗃️ Group   | ⭐ Previous score | ⭐ Current score |                          🔄 Score change                           |
+| :----------------------------------------------------------------- | :---------- | :--------------: | :-------------: | :----------------------------------------------------------------: |
+| [ESLint](https://www.npmjs.com/package/@code-pushup/eslint-plugin) | Suggestions |    🟡 **71**     |      🟡 50      | ![↑ +21.4](https://img.shields.io/badge/%E2%86%91%20%2B21.4-green) |
 
 3 other groups are unchanged.
 
-
 </details>
 
 ## 🛡️ Audits
 
 <details>
 <summary>👍 <strong>3</strong> audits improved</summary>
 
-|🔌 Plugin|🛡️ Audit|📏 Previous value|📏 Current value|🔄 Value change|
-|:--|:--|:--:|:--:|:--:|
-|[ESLint](https://www.npmjs.com/package/@code-pushup/eslint-plugin)|[Require or disallow method and property shorthand syntax for object literals](https://eslint.org/docs/latest/rules/object-shorthand)|🟥 3 warnings|🟩 **passed**|![↓ −100 %](https://img.shields.io/badge/%E2%86%93%20%E2%88%92100%E2%80%89%25-green)|
-|[ESLint](https://www.npmjs.com/package/@code-pushup/eslint-plugin)|[Require braces around arrow function bodies](https://eslint.org/docs/latest/rules/arrow-body-style)|🟥 1 warning|🟩 **passed**|![↓ −100 %](https://img.shields.io/badge/%E2%86%93%20%E2%88%92100%E2%80%89%25-green)|
-|[ESLint](https://www.npmjs.com/package/@code-pushup/eslint-plugin)|[Require `const` declarations for variables that are never reassigned after declared](https://eslint.org/docs/latest/rules/prefer-const)|🟥 1 warning|🟩 **passed**|![↓ −100 %](https://img.shields.io/badge/%E2%86%93%20%E2%88%92100%E2%80%89%25-green)|
+| 🔌 Plugin                                                          | 🛡️ Audit                                                                                                                                | 📏 Previous value | 📏 Current value |                                   🔄 Value change                                    |
+| :----------------------------------------------------------------- | :--------------------------------------------------------------------------------------------------------------------------------------- | :---------------: | :--------------: | :----------------------------------------------------------------------------------: |
+| [ESLint](https://www.npmjs.com/package/@code-pushup/eslint-plugin) | [Require or disallow method and property shorthand syntax for object literals](https://eslint.org/docs/latest/rules/object-shorthand)    |   🟩 **passed**   |  🟥 3 warnings   | ![↓ −100 %](https://img.shields.io/badge/%E2%86%93%20%E2%88%92100%E2%80%89%25-green) |
```

Comment:
🐞 It still swaps the columns for audits.

Reply:
Not my finest moment :sweat_smile: I only fixed https://github.com/code-pushup/cli/pull/755#discussion_r1685793121 for categories, missed audits and groups. Should be properly fixed now.